stdlib/messaging/kafka/src/main/ballerina/src/kafka/kafka_errors.bal

```diff
@@ -29,7 +29,7 @@ public const CONSUMER_ERROR = "{ballerina/kafka}ConsumerError";
 # Represents a Kafka consumer related error
 public type ConsumerError error<CONSUMER_ERROR, Detail>;
 
-# Represents a Kafka producer related error
+# Defines a Kafka producer related error
```

Comment:
Sort of inaccurate to say this defines an error right? Isn't this an error reason?

Reply:
No. This is the error actually. Without this there is no way to distinguish error and producer error.

Reply:
No I meant the constant `PRODUCER_ERROR` itself. It's a string constant. Inaccurate to say defines an error since it's not an error definition

Reply:
Even though this is called *Error Reason*, we use it to *define* the error, right? `{ballerina/kafka}ProducerError` isn't actually an *error reason* IMHO. Instead, we use it to define the different error types. WDYT?